components/org.wso2.carbon.appmgt.usage/org.wso2.carbon.appmgt.usage.publisher/src/main/java/org/wso2/carbon/appmgt/usage/publisher/APIMgtUsageHandler.java

```diff
@@ -356,6 +362,54 @@ public WebApp getWebApp(String context,String version) throws AppManagementExcep
           return webApp;
     }
 
+    /**
+     * Get WebApp for default version web apps.
+     * @param context
+     * @param tenantDomain
+     * @return
+     * @throws AppManagementException
+     */
+    public WebApp getNonVersionedWebApp(String context, String tenantDomain) throws AppManagementException {
+        Connection conn = null;
+        PreparedStatement ps = null;
+        ResultSet rs = null;
```

Comment:
Fixed in https://github.com/wso2/carbon-appmgt/pull/539/commits/d4eba04341eb91c4b68b50cbb78987715b1a657d